doc/release-notes/5.7-release-notes.md

```diff
@@ -0,0 +1,109 @@
+# Dataverse Software 5.7
+
+This release brings new features, enhancements, and bug fixes to the Dataverse Software. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+### Experimental Support for External Vocabulary Services
+
+Dataverse can now be configured to associate specific metadata fields with third-party vocabulary services to provide an easy way for users to select values from those vocabularies. The mapping involves use of external Javascripts. Two such scripts have been developed so far: one for vocabularies served via the [SKOSMOS protocol](https://skosmos.org) and one allowing people to be identified via their [ORCID](https://orcid.org). The guides contain info about the new :CVocConf setting used for configuration and additional information about this functionality. Scripts, examples, and additional documentation are available at the [GDCC GitHub Repository](https://github.com/gdcc/dataverse-external-vocab-support).
```

Comment:
> The guides contain info about the new :CVocConf setting used for configuration and additional information about this functionality.

It would be nice to link to the guides so people can read the additional info more easily.

Reply:
It would be nice! Added in https://github.com/IQSS/dataverse/pull/8128/commits/85e5cf45ccb546d4732035c87b0a79bb3c98f620